src/tokenizer/tokenizer.py

```diff
@@ -721,9 +727,10 @@ def gen_from_string(txt, replace_composite_glyphs=True, replace_html_escapes=Fal
             yield ""
         for w in span.split():
             yield w
```

Comment:
Maybe it is safer to add `if w: yield w`

Reply:
Done!